### DISCUSSION TITLE
Extension/sidebar app for open/closed ports

### DISCUSSION THREAD

**Discussion by lg-nineoneninethreenine** [coder/code-server#3522] [Ideas] [answered]
**FYI: I deployed this on Railway
Can you try to add a section for open ports, because, I have used Gitpod, another cloud editor and it has this port section where, if I am editing my blog and want to keep checking on the blog, it opens a port on its site. Can you try to add it to code-server? It could be helpful if a person is making a node project and want to test it on the cloud or I am checking my Jekyll site.**

**Comment by ghost** [comment, marked as answer]
https://github.com/cdr/code-server/blob/main/docs/FAQ.md#how-do-i-securely-access-web-services
Basically, if your site is running on port 5000, then visit the url at where your instance of code-server is hosted and append `/proxy/5000/`.
Note: Include the trailing `/` at the end. See this for more info https://github.com/cdr/code-server/blob/main/docs/FAQ.md#why-does-the-code-server-proxy-strip-proxyport-from-the-request-path.

Edit: this only covers accessing content, not managing ports

**Reply by lg-nineoneninethreenine** [reply]
Thanks!

**Comment by code-asher** [comment]
We'd like to add some UI around managing ports. Tracking for that is here: https://github.com/cdr/code-server/issues/1944

**Reply by lg-nineoneninethreenine** [reply]
Is it in progress?

**Reply by code-asher** [reply]
Nope, it's currently stalled.

**Reply by lg-nineoneninethreenine** [reply]
Why?

**Reply by jsjoeio** [reply]
Higher-priority issues in front of it + limited resources and time
